core/spec.md

```diff
@@ -597,6 +599,8 @@ form:
 - `"labels": { "STRING": "STRING" * }`
 - `"createdat": "TIMESTAMP"`
 - `"modifiedat": "TIMESTAMP"`
+- `"modelversion": "STRING"`
+- `"compatiblewith": "URI"`
```

Comment:
let's move these to where the other model attributes are defined: https://github.com/xregistry/spec/blob/dfde83550d8662beca8e89766e4be4e6d81d7816/core/spec.md#registry-model

This section is just for xreg metadata (normal user facing) attributes, not model attributes